notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1310_Long_Tracking_Drift_06Apr24.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>Up to you, but I suggest using open circles for these markers in the scatter function, to avoid obscuring each other (I think the option is " facecolors='none' ").</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>